### PR TITLE
Add fabric port isolation attribute

### PR DESCRIPTION
**Pull request by jfeng-arista**
Add fabric port isolation attribute

### CODE REVIEW

**Comment by kcudnik** [comment]
WARNING: line ends in whitespace saiport.h 1888:      * @brief Fabric port isolation setting

**Comment by kcudnik** [comment]
still some issues with build, please fix

**Review by kcudnik — APPROVED** [review]

**Review by rlhui — APPROVED** [review]

**Review by SarathBug — APPROVED** [review]

File: inc/saiport.h [hunk 1]
```diff
@@ -2104,6 +2104,21 @@ typedef enum _sai_port_attr_t
...
     SAI_PORT_ATTR_SUPPORTED_LINK_TRAINING_MODE,
 
+    /**
+     * @brief Fabric port isolation setting.
+     *
+     * true: The link may be enabled in serdes level and the
+     * MAC level, but the link partner will not use
+     * it for traffic distribution.
+     * false: Undo the isolation operation.
+     * This attribute is for fabric links only.
```
Comment (abdosi) [inline inc/saiport.h:2114]: can we add something like this in the comment section. 
Refer sairouterinterface.h

@condition  SAI_PORT_ATTR_TYPE == SAI_PORT_TYPE_FABRIC

**Review by abdosi — CHANGES_REQUESTED** [review]
please check above comment.

**Review by skeesara-nokia — APPROVED** [review]

File: inc/saiport.h (hunk 1 above)
Comment (jfeng-arista) [reply inc/saiport.h:2114]: Hi, 

SAI_PORT_ATTR_TYPE is a read only attribute , and as far as I know that @condition not works for read only attribute.  So checking with you again about if we need to make this change ? 

thank you 

Jie

Comment (jfeng-arista) [reply inc/saiport.h:2114]: I tried add the @condition, and the metadata check failed, 

it looks like based on the following 3 checks, we can not use @condition check for read only attribute. thank you 

https://github.com/opencomputeproject/SAI/blob/f69394f590362b1b02852ec80f00f9a27a02ed97/meta/saisanitycheck.c#L1284
https://github.com/opencomputeproject/SAI/blob/f69394f590362b1b02852ec80f00f9a27a02ed97/meta/saisanitycheck.c#L1455
https://github.com/opencomputeproject/SAI/blob/f69394f590362b1b02852ec80f00f9a27a02ed97/meta/saisanitycheck.c#L1489

**Review by JaiOCP — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Review by kcudnik — APPROVED** [review]

**Comment by rlhui** [comment]
@jfeng-arista  - would you please resolve conflicts?

**Comment by jfeng-arista** [comment]
Hi, 
Can any one help me re-trigger the build for my branch and I could not trigger it by my last comment. 
thank you 
Jie

**Comment by jfeng-arista** [comment]
thank you !

**Pull request merged**